omod/src/main/webapp/fragments/field/locations.gsp

```diff
@@ -3,7 +3,7 @@
     <div>
         <p class="left">
             <label>
-                Please select the location
```

Comment:
Can we drop the "Please select the"? and just use Location?

Reply:
The same applies else where you use please select the location instead of just Location

Reply:
@dkayiwa  I have changed that to location.